nnpdf_data/nnpdf_data/new_commondata/STAR_2012_2JET_A_510GEV/uncertainties.yaml

```diff
@@ -0,0 +1,828 @@
+definitions:
+  stat:
+    description: statistical uncertainty
+    treatment: ADD
+    type: UNCORR
```

Comment:
Are we sure that we have to retain this `stat` uncertainty? Reading the paper, my understanding is that the provided correlation matrices have to be multiplied by the total uncertainty, which is the sum in quadrature of the statistic and systematic uncertainties, for jet-jet, jet-dijet and dijet-dijet.

Reply:
This was indeed not 100% clear for me. 
From [section E](https://arxiv.org/pdf/1906.02740) I read

```
This produces statistical correlations
between the inclusive jet asymmetry measurements that
range from 0.01 for low-pT pairs of inclusive jet bins to
0.04 for high-pT pairs. In contrast, there are no statistical
correlations between the dijet asymmetry measurements.
```

but then:
 
```
The full correlation matrix for the inclusive jet and dijet ALL measurements, including both the statistical and point-to-point systematic uncertainties, is given in the Appendix.
```